foundry/foundry.py

```diff
@@ -365,12 +392,201 @@ def publish(self, foundry_metadata, data_source, title, authors, update=False,
         self.connect_client.add_organization(self.config.organization)
         self.connect_client.set_project_block(
             self.config.metadata_key, foundry_metadata)
-        self.connect_client.add_data_source(data_source)
+
+        # upload via HTTPS if specified
```

Comment:
If this operation is just a wrapper over a method in the connect client, then why aren't we putting the main logic into Connect? I'd imagine people want this capability for MDF in general?

Reply:
good point, waiting on Ben's feedback

Reply:
Ben responded on Slack: yes to moving this over to MDF! I will open a PR in https://github.com/materials-data-facility/connect_client and when that's up, I will close this one